homeassistant/bootstrap.py

```diff
@@ -317,6 +318,11 @@ def async_enable_logging(hass: core.HomeAssistant,
     """
     deps_dir = os.path.join(config_dir, 'deps')
     lib_dir = await async_get_user_site(deps_dir)
-    if lib_dir not in sys.path:
+    if lib_dir in sys.path:
+        return deps_dir
+
+    if run_inside_container():
+        sys.path.append(lib_dir)
```

Comment:
I think that this will cause more harm than good. This will have users install things, not know why it is happening and not being able to repro it outside docker.